### ISSUE TITLE
训练3-mix，loss不下降

### ISSUE THREAD

**Issue by kongcheng-996** [JusperLee/Dual-Path-RNN-Pytorch#65]
24-11-02 22:45:29 [C:\Users\24291\Desktop\Dual-Path-RNN-Pytorch-master\trainer\trainer_Dual_RNN.py:75 - INFO ] Start training from epoch: 8, iter: 0   
C:\Users\24291\Anaconda3\envs\DPRNN\lib\site-packages\torchaudio\extension\extension.py:14: UserWarning: torchaudio C++ extension is not available.
  warnings.warn('torchaudio C++ extension is not available.')
24-11-02 22:45:34 [C:\Users\24291\Desktop\Dual-Path-RNN-Pytorch-master\trainer\trainer_Dual_RNN.py:111 - INFO ] Finished *** <epoch:8, iter:5, lr:1.250e-04, loss:128.000, Total time:0.086 min>
24-11-02 22:45:34 [C:\Users\24291\Desktop\Dual-Path-RNN-Pytorch-master\trainer\trainer_Dual_RNN.py:116 - INFO ] Start Validation from epoch: 8, iter: 0
C:\Users\24291\Anaconda3\envs\DPRNN\lib\site-packages\torchaudio\extension\extension.py:14: UserWarning: torchaudio C++ extension is not available.
  warnings.warn('torchaudio C++ extension is not available.')
24-11-02 22:45:36 [C:\Users\24291\Desktop\Dual-Path-RNN-Pytorch-master\trainer\trainer_Dual_RNN.py:147 - INFO ] Finished *** <epoch:8, iter:4, lr:1.250e-04, loss:120.000, Total time:0.033 min>
24-11-02 22:45:36 [C:\Users\24291\Desktop\Dual-Path-RNN-Pytorch-master\trainer\trainer_Dual_RNN.py:176 - INFO ] No improvement, Best Loss: 120.0000    
24-11-02 22:45:36 [C:\Users\24291\Desktop\Dual-Path-RNN-Pytorch-master\trainer\trainer_Dual_RNN.py:75 - INFO ] Start training from epoch: 9, iter: 0   
C:\Users\24291\Anaconda3\envs\DPRNN\lib\site-packages\torchaudio\extension\extension.py:14: UserWarning: torchaudio C++ extension is not available.
  warnings.warn('torchaudio C++ extension is not available.')
24-11-02 22:45:41 [C:\Users\24291\Desktop\Dual-Path-RNN-Pytorch-master\trainer\trainer_Dual_RNN.py:111 - INFO ] Finished *** <epoch:9, iter:5, lr:1.250e-04, loss:128.000, Total time:0.084 min>
24-11-02 22:45:41 [C:\Users\24291\Desktop\Dual-Path-RNN-Pytorch-master\trainer\trainer_Dual_RNN.py:116 - INFO ] Start Validation from epoch: 9, iter: 0
C:\Users\24291\Anaconda3\envs\DPRNN\lib\site-packages\torchaudio\extension\extension.py:14: UserWarning: torchaudio C++ extension is not available.
  warnings.warn('torchaudio C++ extension is not available.')
24-11-02 22:45:43 [C:\Users\24291\Desktop\Dual-Path-RNN-Pytorch-master\trainer\trainer_Dual_RNN.py:147 - INFO ] Finished *** <epoch:9, iter:4, lr:1.250e-04, loss:120.000, Total time:0.034 min>
24-11-02 22:45:43 [C:\Users\24291\Desktop\Dual-Path-RNN-Pytorch-master\trainer\trainer_Dual_RNN.py:176 - INFO ] No improvement, Best Loss: 120.0000    
24-11-02 22:45:43 [C:\Users\24291\Desktop\Dual-Path-RNN-Pytorch-master\trainer\trainer_Dual_RNN.py:75 - INFO ] Start training from epoch: 10, iter: 0  
C:\Users\24291\Anaconda3\envs\DPRNN\lib\site-packages\torchaudio\extension\extension.py:14: UserWarning: torchaudio C++ extension is not available.
  warnings.warn('torchaudio C++ extension is not available.')
24-11-02 22:45:48 [C:\Users\24291\Desktop\Dual-Path-RNN-Pytorch-master\trainer\trainer_Dual_RNN.py:111 - INFO ] Finished *** <epoch:10, iter:5, lr:1.250e-04, loss:128.000, Total time:0.083 min>
24-11-02 22:45:48 [C:\Users\24291\Desktop\Dual-Path-RNN-Pytorch-master\trainer\trainer_Dual_RNN.py:116 - INFO ] Start Validation from epoch: 10, iter: 0
C:\Users\24291\Anaconda3\envs\DPRNN\lib\site-packages\torchaudio\extension\extension.py:14: UserWarning: torchaudio C++ extension is not available.
  warnings.warn('torchaudio C++ extension is not available.')
24-11-02 22:45:50 [C:\Users\24291\Desktop\Dual-Path-RNN-Pytorch-master\trainer\trainer_Dual_RNN.py:147 - INFO ] Finished *** <epoch:10, iter:4, lr:1.250e-04, loss:120.000, Total time:0.034 min>
Epoch    10: reducing learning rate of group 0 to 6.2500e-05.
24-11-02 22:45:50 [C:\Users\24291\Desktop\Dual-Path-RNN-Pytorch-master\trainer\trainer_Dual_RNN.py:176 - INFO ] No improvement, Best Loss: 120.0000    
24-11-02 22:45:50 [C:\Users\24291\Desktop\Dual-Path-RNN-Pytorch-master\trainer\trainer_Dual_RNN.py:187 - INFO ] Stop training cause no impr for 10 epochs
24-11-02 22:45:50 [C:\Users\24291\Desktop\Dual-Path-RNN-Pytorch-master\trainer\trainer_Dual_RNN.py:191 - INFO ] Training for 10/10 epoches done!
这里为什么loss一直不下降，一直是120，我使用dprnn进行训练

**Comment by JusperLee** [comment]
你的数据是什么呢？
